sodium/src/secbuf.rs

```diff
@@ -30,6 +33,11 @@ impl Bufferable for RustBuf {
         Box::new(RustBuf { b })
     }
 
+    fn from_string(s: String) -> Box<Bufferable> {
+        let b = s.into_bytes().into_boxed_slice();
+        Box::new(RustBuf { b })
```

Comment:
do we still need the `Box::new()` here if the lone above is putting it in a boxed slice?

Reply:
oh, n/m, i see the RustBuf also needs to be boxed